lib/engine/src/trap/frame_info.rs

```diff
@@ -219,6 +219,16 @@ impl Drop for GlobalFrameInfoRegistration {
     }
 }
 
+/// Represents a continuous region of executable memory starting with a function
+/// entry point.
+pub struct FunctionExtent {
```

Comment:
`#[derive(Debug)]` please!

Reply:
Does this need to be `repr(C)`? 🤔

Reply:
I don't think it needs to be repr(C) right now, but I'd like to use FunctionBodyPtr in the VM types so I'll go ahead and add it now.